app/components/grad_culminating_experiences_dropdowns_component.rb

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+class GradCulminatingExperiencesDropdownsComponent < ApplicationComponent
+  attr_reader :form,
+              :resource
+
+  def initialize(form:, resource:)
+    @form = form
+    @resource = resource
+  end
+
+  def sub_work_type_dropdown
+    if resource.work_type == Work::Types.professional_doctoral_culminating_experience
+      [
+        'Capstone Project',
+        'Culminating Research Project',
+        'Doctor of Nursing Practice Project',
+        'Integrative Doctoral Research Project',
+        'Praxis Project',
+        'Public Performance'
+      ].freeze
```

Comment:
I didn't add these to a yml because right now there isn't any use for them outside of ScholarSphere, nor do we have a system where we could potentially pull them from.  Ideally, we'd like to pull the program and degree controlled vocabularies directly from ETDA, but we don't have an endpoint for this.  We can use the `QA` gem to set up this endpoint (https://github.com/samvera/questioning_authority).  We have the `QA` gem set up in ScholarSphere, so pulling from the ymls is a temporary solution for controlled vocabularies that can hopefully be pulled from ETDA in the future.